### PR TITLE
Add Microsoft Game Development Kit

### PR DESCRIPTION
**Pull request by diedyesterdaywashere**
### SteamDB app page links to a few games using this

https://steamdb.info/app/1657630/depots/

### Brief explanation of the change

added a rule for GDK

### CODE REVIEW

File: descriptions/SDK.Game_Development_Kit.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The [Microsoft Game Development Kit](https://learn.microsoft.com/en-us/gaming/gdk/) (GDK) is a complete set of tools, APIs, extensions, and programing models that can be used across current and future Microsoft Gaming platforms and initiatives.
```
Comment (Lifeismana) [inline descriptions/SDK.Game_Development_Kit.md:1]:
```suggestion
The [**Microsoft Game Development Kit (GDK)**](https://learn.microsoft.com/en-us/gaming/gdk/) is a complete set of tools, APIs, extensions, and programing models that can be used across current and future Microsoft Gaming platforms and initiatives.
```

File: rules.ini [hunk 2]
```diff
@@ -185,6 +185,7 @@ Discord = (?:^|/)(?:lib)?discord(?:|-rpc|_game_sdk)\.(?:dll|dylib|so)$
 EpicOnlineServices = (?:^|/)(?:lib)?eossdk
 FMOD = (?:^|/)(?:lib)?fmod(?:l|ex|exl|studio|studiol)?(?:64)?\.(?:dylib|dll|so)$
 FreeImage = (?:^|/)FreeImage(?:\.so|\.dll)$
+Game_Development_Kit = (?:^|/)(?:XGamingRuntime|XGamingRuntimeThunks|XCurl|Microsoft_Xbox_Services_141_GDK_C_Thunks)\.dll$
```
Comment (Lifeismana) [inline rules.ini:188]:
```suggestion
Game_Development_Kit = (?:^|/)(?:XGamingRuntime(?:Thunks)?|XCurl|Microsoft_Xbox_Services_141_GDK_C_Thunks)\.dll$
```

**Review by Lifeismana — CHANGES_REQUESTED** [review]
also run the generateTestStrings script

**Comment by diedyesterdaywashere** [comment]
> also run the generateTestStrings script

could you please run it for me? i dont have php installed locally so i can't do that...

or maybe y'all should make an action that would run the generateteststrings thing on pr creation?

**Pull request merged**